frontend/src/views/WheresMyGeneV2/components/HeatMap/index.tsx

```diff
@@ -277,13 +277,13 @@ export default memo(function HeatMap({
                * which is an error for echarts
                */
               if (!selectedGeneData?.length) {
-                const height =
-                  document.getElementById(`y-axis-${tissue.name}`)
-                    ?.clientHeight ?? 0;
+                // const height =
+                //   document.getElementById(`y-axis-${tissue.name}`)
+                //     ?.clientHeight ?? 0;
                 return (
                   <div
                     key={`y-axis-${tissue.name}`}
-                    style={{ height: `${height + X_AXIS_CHART_HEIGHT_PX}px` }}
+                    style={{ height: `${0}px` }} // This should be investigated more as to why it fixes dot alignment
```

Comment:
Hmm seems like this was added 2 weeks ago here to fix a [different bug](https://github.com/chanzuckerberg/single-cell-data-portal/issues/4870) in current WMG (not WMG2):

https://github.com/chanzuckerberg/single-cell-data-portal/pull/4871/files

My assumption is that `{height + X_AXIS_CHART_HEIGHT_PX}p` will create an empty div with height that matches the tissue block's height, so it solves the misalignment bug that happens when a tissue's heatmap is not rendered due to not having associated gene data. 

Maybe the height gap between tissue is different in WMG2 or something, so it's misaligned again?

CC: @atarashansky in case you have some ideas 😆 

Thank you!

Reply:
Ah Alec is out for the rest of the week and will be back on Monday lol

Can you wait for him to be back on Monday? Otherwise I can take a look later today!